catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/history/HistorianTest.java

```diff
@@ -162,7 +162,7 @@ public void testUpdateResponseSkipProperty() throws SourceUnavailableException,
     verifyZeroInteractions(catalogProvider);
   }
 
-  @Test
+  @Test(expected = NullPointerException.class)
```

Comment:
We should eliminate the potential for an NPE instead of accepting it as normal behavior